backend/jobs/api/employer.py

```
@@ -13,7 +13,7 @@ class ApplicantPerJobView(generics.ListAPIView):
     model = Applicant
     serializer_class = ApplicantDetailSerializer
 
-    # @method_decorator(user_is_employer)
+    @method_decorator(user_is_employer)
     def dispatch(self, request, *args, **kwargs):
```

Ruff (0.13.3)

Unused method argument: `request`

(ARG002)